plugins/org.eclipse.fordiac.ide/src/org/eclipse/fordiac/ide/issuereport/GitIssueCreator.java

```
@@ -121,18 +120,21 @@ private static Optional<String> createGitLabIssue(final IssueInfo info) {
 		final String accessToken = PreferenceConstants.getReportGitLabToken();
 		final String labels = String.join(",", info.labels()); //$NON-NLS-1$
 
-		final String uri = baseURI + "/api/v4/projects/%s/issues?title=%s&description=%s&labels=%s"; //$NON-NLS-1$
-
-		final String reportingURI = uri.formatted(URLEncoder.encode(projectPath, StandardCharsets.UTF_8),
+		final String uri = "%s/api/v4/projects/%s/issues?title=%s&description=%s&labels=%s"; //$NON-NLS-1$
+		final String reportingURI = uri.formatted(baseURI, URLEncoder.encode(projectPath, StandardCharsets.UTF_8),
 				URLEncoder.encode(info.title(), StandardCharsets.UTF_8),
 				URLEncoder.encode(info.body(), StandardCharsets.UTF_8),
 				URLEncoder.encode(labels, StandardCharsets.UTF_8));
-
 		try {
 			final HttpRequest request = HttpRequest.newBuilder().uri(URI.create(reportingURI))
 					.header("PRIVATE-TOKEN", accessToken) //$NON-NLS-1$
 					.POST(HttpRequest.BodyPublishers.noBody()).build();
-			return makeRequest(request);
+			final Optional<String> body = makeRequest(request);
+			if (body.isEmpty()) {
+				return body;
+			}
+			final Gson gson = new Gson();
```

This should be moved outside the `try`-block as you might hide exceptions from Gson otherwise